api/groups/blockGroup.go

```diff
@@ -234,26 +232,35 @@ func parseBlockQueryOptions(c *gin.Context) (api.BlockQueryOptions, error) {
 	return options, nil
 }
 
-func parseAlteredAccountsForBlockQueryOptions(c *gin.Context) (string, bool, error) {
+func parseAlteredAccountsForBlockQueryOptionsWithoutRequestType(c *gin.Context) (api.GetAlteredAccountsForBlockOptions, error) {
```

Comment:
[comment]

Why not return pointer to `api.GetAlteredAccountsForBlockOptions` ?

Reply:
Since all the mechanism relies of those options as value, not as pointer, I would leave it as it is